src/sonic-config-engine/portconfig.py

```diff
@@ -26,8 +26,7 @@
 INTF_KEY = "interfaces"
 OPTIONAL_HWSKU_ATTRIBUTES = ["fec", "autoneg"]
 
-BRKOUT_PATTERN = r'(\d{1,3})x(\d{1,3}G)(\[\d{1,3}G\])?(\((\d{1,3})\))?'
-
+BRKOUT_PATTERN = r'(\d{1,3})x(\d{1,3}G)(\[(\d{1,3}G,?)*\])?(\((\d{1,3})\))?'
```

Comment:
Is this to support more alternative speeds?  If so, do we have more unit test cases to cover this?

Reply:
Yes,  you are right. To cover this, I will update one of  the samples in `src/sonic-config-engine/tests/sample_platform.json`

Reply:
#updated